gradle/publish-jar.gradle

```diff
@@ -53,3 +66,14 @@ publishing {
 tasks.withType(GenerateModuleMetadata) {
     enabled = false
 }
+
+def hasSigningKey = project.findProperty("signingKey") && project.findProperty("signingKeyId")
+if(hasSigningKey) {
+    signing {
+        // Expect that the gpg key is base64 decoded as a secret in CircleCI
+        // to avoid problems with newlines in CircleCI env vars
+        useInMemoryPgpKeys(findProperty("signingKeyId"), new String(findProperty("signingKey").decodeBase64()), findProperty("signingPassword"))
```

Comment:
Instead of `findProperty($Name)`, would it be more clear to use `System.env.$NAME`?

Reply:
Yeah, I was a bit confused where this is coming from? Environment var is probably clearer.